timeline/tests/test_templatetags.py

```diff
@@ -180,6 +180,21 @@ def test_get_app_icon_html_plugin(self):
         self.assertIn(plugin.title, ret)
         self.assertIn(url, ret)
```

Comment:
As said before, check the URLs in these tests explicitly to ensure we get the appropriate URL. Do this with:
1. Event created with `app=projectroles`
2. Event created with some other project app (`example_project_app` is a good choice)
3. Event created with a site app (`example_site_app` is recommended :))

Then assert the URL returned corresponds to whatever `reverse()` returns with the related `entry_point_url_id` and valid kwargs. In case of projectroles the entry point is hardcoded to `projectroles:detail`.